### ISSUE TITLE
SCM - option to prefer repo name

### ISSUE THREAD

**Issue by klmcwhirter** [microsoft/vscode#113401]
Option to prefer repo name to branch name in Source Control Repository view.
Please refer to https://github.com/microsoft/vscode/issues/104801#issuecomment-684589792

This design choice makes the git integration unusable for many of us.

![image](https://user-images.githubusercontent.com/1332462/103112910-05c13780-460d-11eb-9541-f35f1cb0a0a0.png)

Versus

![image](https://user-images.githubusercontent.com/1332462/103112929-17a2da80-460d-11eb-9aa2-d43a9d9f6d5f.png)

**Comment by eamodio** [comment]
I agree that this is an issue, unfortunately due to implementation complexities with how the branch name is displayed (it's a command) it is tricky to balance this.

One option might be to add a setting or something to limit the number of characters that will be shown in the branch name (this is still not ideal, as you would really only want it if there isn't space).

I'm also open to other options to deal with this, given the current constraints.

**Comment by lszomoru** [comment]
We have made improvements in this area and the Repositories view now always shows the repository name.
